common/src/test/kotlin/json/MessageTest.kt

```diff
@@ -76,7 +76,7 @@ fun `User serialization`() {
         id shouldBe "334385199974967042"
         pinned shouldBe false
         editedTimestamp shouldBe null
-        author?.let {
+        author.let {
```

Comment:
With `author` being null-safe there is no need for the let scope.

Reply:
Sure, I can turn that into a `with` statement instead.